releases/release-1.31/release-notes/maps/pr-125641-map.yaml

```diff
@@ -0,0 +1,9 @@
+pr: 125641
+releasenote:
+  text: "$ kubectl ns -h\nView or set the current namespace\n\nUsage:\n  kubectl ns
```

Comment:
This is another parsing issue because the author didn't include a release note: https://github.com/kubernetes/kubernetes/pull/125641